05-GitBest.Rmd

```diff
@@ -15,53 +15,58 @@ You can host multiple projects on GitHub at the same time but it would be a mist
 
 ## Use branches to avoid heartache {#Best5.2}
 
-A `branch` is a way to develop scripts (e.g., a new feature or modification) alongside functions scripts, while not affecting the original (i.e., the main or default branch). It is good practice to work on short-lived branches.If you are working on developing new code, you will probably work from a single branch, called the main or default branch, and commit changes as you go to the origin on Github. But once you have a functional amount of code, or are collaborating on code development, you will want to create a `branch` so that you are not risking breaking what is already working!
```

Comment:
This first sentence isn't accurate as you can't use function scripts on another branch while working on a different branch. You can't access anything from another branch as they're kept completely separate. Does my edit make sense?